src/Microsoft.AspNetCore.Blazor.Build/ReferenceFromSource.props

```diff
@@ -7,9 +8,23 @@
   
   This is only intended for use by other projects in this repo.
   -->
-  
+  <PropertyGroup>
+    <BlazorBuildReferenceFromSource>true</BlazorBuildReferenceFromSource>
+    <BlazorMonoRuntimeBasePath>$(MSBuildThisFileDirectory)../mono/</BlazorMonoRuntimeBasePath>
+    <MonoLinkerPath>$(BlazorMonoRuntimeBasePath)tools/binaries/illink/illink.dll</MonoLinkerPath>
+    <MonoBaseClassLibraryPath>$(BlazorMonoRuntimeBasePath)dist/optimized/bcl/</MonoBaseClassLibraryPath>
+    <MonoBaseClassLibraryFacadesPath>$(BlazorMonoRuntimeBasePath)dist/optimized/bcl/Facades/</MonoBaseClassLibraryFacadesPath>
+    <MonoAsmjsRuntimePath>$(BlazorMonoRuntimeBasePath)dist/optimized/asmjs/</MonoAsmjsRuntimePath>
+    <MonoWasmRuntimePath>$(BlazorMonoRuntimeBasePath)dist/optimized/wasm/</MonoWasmRuntimePath>
+    <BlazorJsPath>$(MSBuildThisFileDirectory)../Microsoft.AspNetCore.Blazor.Browser.JS/dist/blazor.js</BlazorJsPath>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)targets\All.props" />
   <Import Project="$(MSBuildThisFileDirectory)targets\All.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)../mono/mono.targets" />    
```

Comment:
nit, pick a kind of *slash* and stick to it. I think generally the MSBuild style is to the use backslash.